ioctl/newcmd/alias/alias.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2019 IoTeX Foundation
+// This is an alpha (internal) release and is not suitable for production. This source code is provided 'as is' and no
+// warranties are given as to title or non-infringement, merchantability or fitness for purpose and, to the extent
+// permitted by law, all liability for your use of the code is disclaimed. This source code is governed by Apache
+// License 2.0 that can be found in the LICENSE file.
+
+package alias
```

Comment:
can you introduce the background and purpose of this PR?
it looks like the code is duplicate of existing alias.go

Reply:
please reference issue#1768, and the alias.go in ioctl\newcmd is used for the same reason of the bc.go in ioctl\newcmd. This PR is about migrating old ioctl cmd to newcmd.

Reply:
should remove the old alias.go?